ckanext/canada/controller.py

```diff
@@ -458,11 +458,18 @@ def output_feed(self, results, feed_title, feed_description,
             last_page=navigation_urls['last'],
         )
 
+        chromos = h.recombinant_get_types()
+
         for pkg in results:
+            title, notes = None, None
+            if pkg['type'] in chromos:
+                chromo = h.recombinant_get_chromo(pkg['type'])
+                title= _(chromo['title'])
+                notes = _(pkg['notes'])
```

Comment:
we translate it here https://github.com/open-data/ckanext-canada/blob/master/ckanext/canada/templates/public/snippets/package_item.html#L10